cosmo_tester/framework/test_hosts.py

```diff
@@ -436,6 +432,87 @@ def wait_for_manager(self):
                     )
 
 
+class _CloudifyDatabaseOnly(_CloudifyManager):
+    """
+    This class represents an instance of a Cloudify Database without a manager
+
+    Most of the inherited functions here are to avoid any incorrect usage of
+    the class since most of these functions rely on the manager existing on the
+    machine.
+    """
+
+    def __init__(self):
+        super(_CloudifyDatabaseOnly, self).__init__(upload_plugins=False)
+
+    def __str__(self):
+        return 'Cloudify Manager - database only VM ({image}) [{index}:{ip}]' \
+            .format(image=self.image_name,
+                    index=self.index,
+                    ip=self.ip_address, )
+
+    @property
+    def branch_name(self):
+        raise NotImplementedError()
+
+    def create(
+            self,
+            index,
+            public_ip_address,
+            private_ip_address,
+            networks,
+            rest_client,
+            ssh_key,
+            cfy,
+            attributes,
+            logger,
+            tmpdir
+    ):
+        self.index = index
+        self.ip_address = public_ip_address
+        self.private_ip_address = private_ip_address
+        self.client = rest_client
+        self.deleted = False
+        self._ssh_key = ssh_key
+        self._cfy = cfy
+        self._attributes = attributes
+        self._logger = logger
+        self._tmpdir = os.path.join(tmpdir, str(uuid.uuid4()))
+        os.makedirs(self._tmpdir)
+        self.additional_install_config = {}
+
+    @retrying.retry(stop_max_attempt_number=6 * 10, wait_fixed=10000)
+    def verify_services_are_running(self):
+        with self.ssh() as fabric_ssh:
+            # validate PostgreSQL server is running
+            try:
+                fabric_ssh.sudo('su -c "psql -l" postgres')
+                return True
+            except Exception:
+                self._logger.warn(
+                    'PostgreSQL is not in an active state. Retrying...')
```

Comment:
show the actual exception please, to limit guesswork when reading the logs, eg:
```python
except Exception as e:
    self._logger.warn('PostgreSQL is not in an active state. Retrying... (%s)', e)
```

Reply:
Agreed, done.